### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by nikhiljugale007**
Added home page/landing page for e-commerce site for footware.

### CODE REVIEW

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]
For .img-round you can use object-fit: cover.
Other things looks fine 👍

File: index.css [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+	box-shadow: 0 5px 3px -3px var(--box-shadow);
+}
+
+.header-sub-container {
+	display: flex;
+	align-items: center;
+	gap: 0 2rem;
+}
+.btn {
+	color: #2d3748;
```
Comment (amarphule) [inline index.css]: you can use variable for color

**Review by amarphule — COMMENTED** [review]
Looks good, for a button you can use `cursor:pointer`. and different text color text.
nik

File: index.css (hunk 1 above)
Comment (nikhiljugale007) [reply index.css]: Implemented suggestion in the latest commit.

**Comment by nikhiljugale007** [comment]
> For .img-round you can use object-fit: cover. Other things looks fine 👍

added object-fit:cover property.

**Pull request merged**